src/components/badge/badge.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Badge from "./badge.component";
+
+const renderComponent = (props = {}) =>
+  render(<Badge {...props}>Button</Badge>);
```

Comment:
**comment**: just wanted to mention, this simple abstraction is alright for now, but let's keep an eye out for similar ones in future PRs, especially when we're dealing with more complex test files.

It's important to ensure that any helper functions we add genuinely enhance clarity and don't make the file structure more complicated in the name of staying DRY.